docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,763 @@
+      # and apply any new model configuration, to also force a subsequent
+      # roll of your domain's WebLogic Server pods.
+      restartVersion: '1'
+
+      configuration:
+
+        # Settings for domainHomeSourceType 'FromModel'
+        model:
+          # Valid model domain types are 'WLS', 'JRF', and 'RestrictedJRF', default is 'WLS'
+          domainType: "WLS"
+
+          # Optional configmap for additional models and variable files
+          #configMap: sample-domain1-wdt-config-map
+
+          # All 'FromModel' domains require a runtimeEncryptionSecret with a 'password' field
+          runtimeEncryptionSecret: sample-domain1-runtime-encryption-secret
+
+        # Secrets that are referenced by model yaml macros
+        # (the model yaml in the optional configMap or in the image)
+        #secrets:
+        #- sample-domain1-datasource-secret
+  ```
+  {{% /expand %}}
+
+
+  > **Note**: Before you deploy the domain custom resource, determine if you have Kubernetes cluster worker nodes that are remote to your local machine. If so, you need to put the Domain's image in a location that these nodes can access and you may also need to modify your Domain YAML file to reference the new location. See [Ensuring your Kubernetes cluster can access images]({{< relref "/samples/simple/domains/model-in-image/_index.md#ensuring-your-kubernetes-cluster-can-access-images" >}}).
+
+Run the following command to create the domain custom resource:
+
+```bash
+$ kubectl apply -f /tmp/mii-sample/domain-resources/WLS/mii-initial-d1-WLS-v1.yaml
+```
+> **Note**: If you are choosing not to use the predefined Domain YAML file and instead created your own Domain YAML file earlier, then substitute your custom file name in the above command. Previously, we suggested naming it `/tmp/mii-sample/mii-initial.yaml`.
+
+Verify the WebLogic server pods are all running
+
+```
+$ kubectl get all -n sample-domain1-ns
+NAME                                 READY   STATUS    RESTARTS   AGE
+pod/sample-domain1-admin-server      1/1     Running   0          41m
+pod/sample-domain1-managed-server1   1/1     Running   0          40m
+pod/sample-domain1-managed-server2   1/1     Running   0          40m
+
+NAME                                       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/sample-domain1-admin-server        ClusterIP   None           <none>        7001/TCP   41m
+service/sample-domain1-cluster-cluster-1   ClusterIP   100.66.99.27   <none>        8001/TCP   40m
+service/sample-domain1-managed-server1     ClusterIP   None           <none>        8001/TCP   40m
+service/sample-domain1-managed-server2     ClusterIP   None           <none>        8001/TCP   40m
+
+
+```
+
+#### Invoke the web application
+
+##### Create loadbalancer
+
+Create loadbalancer to access WebLogic server admin console and to access applications deployed in cluster.
+Tanzu supports mettallb loadbalancer and nginx ingress for routing.
```

Comment:
supports mettallb loadbalancer-> supports the MetalLB load balancer
nginx  -> NGINX